src/components/Button/Button.props.ts

```diff
@@ -126,6 +126,7 @@ export type ButtonProps = {
    * - "outlined": medium-emphasis button with a stroke around the button container and no fill,
    *   typically used for secondary actions;
    * - "ghost": transparent button with no stroke, typically used for links.
```

Comment:
Please improve this doc line, why should ghost be typically used for links now that there's the link type

Reply:
Because there are cases where we want to preserve margin and padding of a button, but show the content as a link so without border and background color

Reply:
the doc still doesn't make sense, it suggests using this for links while there is a link type. if there are use cases for this to be used as link as well we have to describe them; otherwise let's remove the note.

Reply:
ok, it is better to remove the recommendation